repo/packages/C/confluent-connect/2/resource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "assets": {
+    "container": {
+      "docker": {
+        "image": "dbtucker/cp-kafka-connect"
```

Comment:
For now, the tag is a separate value.   The image specification in marathon.json.mustache is built from this value and then the docker-tag.